message.rb

```diff
@@ -0,0 +1,13 @@
+require './morse_translator'
+
+class Message
+  attr_reader :text, :translator
+  def initialize(text, morse_translator)
+    @text = text
+    @translator = morse_translator
+  end
+
+  def translation
+    morse_translator(@text)
```

Comment:
`morse_translator` is the name of the variable you passed into your `initialize` method. If you want to access that translator, you need to call `@translator` (the name of your instance variable) or `translator` (the name of the method you get from your `attr_reader`) here.